gallery/lib/home.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+
+import 'demo.dart';
+import 'settings.dart';
+import 'splash.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Gallery'),
+        leading: IconButton(
+          icon: Icon(Icons.color_lens),
+          onPressed: () {
+            Navigator.of(context).push(
+              MaterialPageRoute(
+                fullscreenDialog: true,
```

Comment:
why are these dialogs?

Reply:
This flag actually makes the route slide up from the bottom and have an X in the app bar, not really a dialog in the traditional sense